src/models/decisions/decision.ts

```diff
@@ -0,0 +1,17 @@
+import { EnumValue } from "../../helpers/advancedTypes";
+
+export enum VerificationDecision {
+    FALSE = "false",
+    TRUE = "true",
+    UNSURE = "unsure",
+    SKIP = "skip",
+}
+
+// A polymorphic structure that can be used to represent any decision type
+// that implements the interface
+export interface IDecision {
+    confirmed: EnumValue<VerificationDecision>;
+    identifier: symbol;
```

Comment:
what is this identifying?

Reply:
Strangely, it identifies where a decision came from (e.g. what decision button the decision model was created in)

Since we're leaning into `decisionId` (`origin`) uniquely identifying where a decision came from, I'm thinking that I'll remove `identifier` and use `decisionId` in its place